doc/radosgw/admin.rst

```diff
@@ -100,6 +100,13 @@ For example::
   radosgw-admin subuser create --uid=johndoe --subuser=johndoe:swift --access=full
 
 
+Also you can create subuser without specifying ``--uid={username}``. ::
```

Comment:
Can you rephrase it so, 
It is preferred to be explicit when creating subusers, specifying users with a `--uid`. If a `--uid` is not supplied, we parse the `uid` until the last colon. 

<syntax>

Reply:
Sure.